initfiles/componentfiles/configschema/xsd/esp_service_smc.xsd

```diff
@@ -13,6 +13,19 @@
             <xs:attribute name="warnIfFreeMemoryUnder" type="xs:nonNegativeInteger" use="optional" default="5" hpcc:displayName="Warn Free Memory" hpcc:group="Monitoring" hpcc:tooltip="Available memory under this value is flagged as warning in monitoring output"/>
         </xs:attributeGroup>
 
+        <xs:attributeGroup name="elk">
+            <xs:attribute name="integrateKibana" type="xs:boolean" use="required"  hpcc:defaultInCode="true" hpcc:tooltip="Declare if Kibana dashboard to be integrated in ECLWatch"/>
+            <xs:attribute name="kibanaAddress" type="xs:string" use="optional" hpcc:defaultInCode="localhost" hpcc:tooltip="Address (ip/hostname) of Kibana server"/>
+            <xs:attribute name="kibanaPort" type="xs:nonNegativeInteger" use="optional" hpcc:defaultInCode="5601" hpcc:tooltip="Port number of target Kibana server"/>
```

Comment:
nonNegativeInteger is fine. I thought we were going to have more types available such as port, ip, etc?

Reply:
If we want to add a "port" type we should. It should be a different JIRA so that all XSDs are changed as well as adding the type. The JIRA should add some ranges for valid port numbers, say min of 1000 and max of 10000.